entrypoint.sh

```diff
@@ -98,6 +99,8 @@ host="https://$SHOP_STORE"
 # Use the $SHOP_PASSWORD defined as a Github Secret for password protected stores.
 [[ -z ${SHOP_PASSWORD+x} ]] && shop_password='' || shop_password="$SHOP_PASSWORD"
 
+theme_root="$THEME_ROOT"
```

Comment:
Minor minor nit: This should be `${THEME_ROOT:-.}` so that we default to `.` here as well. This would make it so that someone who tries to copy this code and run it in CircleCI wouldn't have to think about it.

Reply:
Erh no need to have _you_ do this for me. It's more time wasted on both ends. It's a minor nit, I can fix it :P